contrib/emptty/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "emptty"
+pkgver = "0.12.1"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+makedepends = ["libx11-devel", "linux-pam-devel"]
+pkgdesc = "Dead simple CLI Display Manager on TTY"
+maintainer = "Michal Tvrznik <emporeor@gmail.com>"
+license = "MIT"
+url = "https://github.com/tvrzna/emptty"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "0220230cdee3dedd7ccd6f2d81c57b29e2169159c8e28263d58abaf20093ce23"
+env = {"CGO_ENABLED": "1"}
```

Comment:
```suggestion
make_env = {"CGO_ENABLED": "1"}
```

(and put it above where it goes)

Reply:
Renamed and moved.